lib/src/controls/navigation/tree_view.dart

```diff
@@ -727,50 +752,69 @@ class _TreeViewItem extends StatelessWidget {
                           },
                         ),
                       ),
+
+                    // Expand icon with hitbox
                     if (item.isExpandable)
                       if (item.loading)
                         item.loadingWidget ?? loadingWidgetFallback
                       else
                         GestureDetector(
                           behavior: HitTestBehavior.deferToChild,
                           onTap: onExpandToggle,
-                          child: Icon(
-                            item.expanded
-                                ? FluentIcons.chevron_down
-                                : direction == TextDirection.ltr
-                                    ? FluentIcons.chevron_right
-                                    : FluentIcons.chevron_left,
-                            size: 8.0,
-                            color: Colors.grey[80],
+                          child: Container(
+                            // The hitbox for the chevron is three times the
+                            // chevron's (max) width.
+                            width: 24,
+                            // The hitbox fills the available height.
+                            height: double.infinity,
+                            decoration: BoxDecoration(
+                              color: Colors.transparent,
+                              borderRadius: BorderRadius.circular(5.0),
+                            ),
+                            child: Icon(
+                              item.expanded
+                                  ? FluentIcons.chevron_down
+                                  : direction == TextDirection.ltr
+                                      ? FluentIcons.chevron_right
+                                      : FluentIcons.chevron_left,
+                              size: 8.0,
+                              color: Colors.grey[80],
+                            ),
                           ),
                         )
-                    else if (item._anyExpandableSiblings)
-                      // if some child items are expandable and others are not,
-                      // make sure that they line up vertically the same for the
-                      // same depth
+                    else
+                      // Add padding to make all items of the same depth level
+                      // have the same indentation, regardless whether or not
+                      // they are expandable.
```

Comment:
Can you explain this better? I don't get it

Reply:
I should have added a screenshot to illustrate:
![TreeView_indentation](https://user-images.githubusercontent.com/103884835/188910324-29a1fbfd-daa4-41bf-abd1-f37581fd7068.jpg)
As you can see with the green line, all items with the same parent have the same indentation, no matter if they are expandable or not. But items of the same level, but with different parents, don't get the same indentation but show an offset (red line).

Reply:
Got it!